v0/destinations/drip/transform.js

```diff
@@ -0,0 +1,238 @@
+const { EventType } = require("../../../constants");
+const {
+  getDestinationExternalID,
+  getFieldValueFromMessage,
+  constructPayload,
+  extractCustomFields,
+  isEmptyObject,
+  CustomError,
+  removeUndefinedAndNullValues,
+  defaultRequestConfig,
+  defaultPostRequestConfig,
+  getErrorRespEvents,
+  getSuccessRespEvents
+} = require("../../util");
+const logger = require("../../../logger");
+const {
+  ENDPOINT,
+  identifyMapping,
+  trackMapping,
+  campaignMapping,
+  IDENTIFY_EXCLUSION_FIELDS,
+  TRACKING_EXLCUSION_FIELDS
+} = require("./config");
+const {
+  idValidity,
+  isValidEmail,
+  isValidTimestamp,
+  callValidity,
+  createUpdateUser
+} = require("./util");
+
+const identifyResponseBuilder = async (message, { Config }) => {
+  const { accountId } = Config.accountId;
+
+  const id = getDestinationExternalID(message, "dripId");
```

Comment:
if user is passing externalId, I think we do not need to explicitly check from transformer if that user actually exists. For externalId, we can just proceed with normal identify flow for existing user, i.e update the user traits. But have to check first, whether the destination sends a proper error response in such case. Let's discuss this later.